# disable tests apparently affected by ImageMagick regressions. see
# https://gitlab.mister-muffin.de/josch/img2pdf/issues/178 and
# https://github.com/ImageMagick/ImageMagick6/issues/285 for related bug
# reports.
--- a/src/img2pdf_test.py
+++ b/src/img2pdf_test.py
@@ -5590,7 +5590,7 @@ def test_jpg_2000(tmp_path_factory, jpg_2000_img, jpg_2000_pdf):
     reason="test utilities not available on Windows and MacOS",
 )
 @pytest.mark.skipif(
-    not HAVE_JP2, reason="requires imagemagick with support for jpeg2000"
+    True, reason="affected by ImageMagick regressions"
 )
 def test_jpg_2000_rgba8(tmp_path_factory, jpg_2000_rgba8_img, jpg_2000_rgba8_pdf):
     tmpdir = tmp_path_factory.mktemp("jpg_2000_rgba8")
@@ -5605,7 +5605,7 @@ def test_jpg_2000_rgba8(tmp_path_factory, jpg_2000_rgba8_img, jpg_2000_rgba8_pdf
     reason="test utilities not available on Windows and MacOS",
 )
 @pytest.mark.skipif(
-    not HAVE_JP2, reason="requires imagemagick with support for jpeg2000"
+    True, reason="affected by ImageMagick regressions"
 )
 def test_jpg_2000_rgba16(tmp_path_factory, jpg_2000_rgba16_img, jpg_2000_rgba16_pdf):
     tmpdir = tmp_path_factory.mktemp("jpg_2000_rgba16")
@@ -6412,8 +6412,8 @@ def test_miff_cmyk8(tmp_path_factory, miff_cmyk8_img, tiff_cmyk8_img, miff_cmyk8
 
 
 @pytest.mark.skipif(
-    sys.platform in ["win32"],
-    reason="test utilities not available on Windows and MacOS",
+    True,
+    reason="affected by ImageMagick regressions",
 )
 def test_miff_cmyk16(
     tmp_path_factory, miff_cmyk16_img, tiff_cmyk16_img, miff_cmyk16_pdf
